Controls: Remove ASP.NET references (#2420)
DevExpress/devextreme-documentation: concepts/05 UI Components/Popover/05 Customize the Appearance/05 Customize the Content/1 Specifying the Content Template.md

> The template implementation depends on the used framework or library. Examples of jQuery, and Angular are presented below. They show how to create a template consisting of static (text) and dynamic (the [Switch](/concepts/05%20UI%20Components/Switch/00%20Overview.md '/Documentation/Guide/UI_Components/Switch/Overview/') UI component) content. 

Reviewer: ```suggestion
The following code shows how to create a template consisting of static (text) and dynamic (the [Switch](/concepts/05%20UI%20Components/Switch/00%20Overview.md '/Documentation/Guide/UI_Components/Switch/Overview/') UI component) content:
```
